Dockerfile

```diff
@@ -1,6 +1,17 @@
+FROM golang:1.12 AS build-image
+
+ADD . /go/src/github.com/kubeflow/pytorch-operator
+
+WORKDIR /go/src/github.com/kubeflow/pytorch-operator
+
+# Build pytorch operator v1beta2 binary
+RUN go build cmd/pytorch-operator.v1beta2
+# Build pytorch operator v1 binary
+RUN go build cmd/pytorch-operator.v1
```

Comment:
RUN go build ./cmd/pytorch-operator.v1
RUN go build ./cmd/pytorch-operator.v1beta2

Reply:
Yeah we need `./` here.